Facebook/build/resources/main/META-INF/persistence.xml

```diff
@@ -5,12 +5,13 @@
 
     <persistence-unit name="Facebook" transaction-type="RESOURCE_LOCAL">
         <class>org.vso.models.data.User</class>
+        <class>org.vso.models.data.FriendShip</class>
         <class>org.vso.models.data.Post</class>
         <properties>
             <property name="jakarta.persistence.jdbc.driver" value="org.postgresql.Driver" />
             <property name="jakarta.persistence.jdbc.url" value="jdbc:postgresql://localhost:5432/Facebook" />
             <property name="jakarta.persistence.jdbc.user" value="postgres" />
-            <property name="jakarta.persistence.jdbc.password" value="admin" />
+            <property name="jakarta.persistence.jdbc.password" value="Bibs7980" />
```

Comment:
If you change the password for the database in the main code the other people needs to change their passwords for the database